oracle/README.md

```diff
@@ -7,13 +7,13 @@ Tests are ran against our own test vectors in `tests/test_vectors`.
 You can re-generate the test vectors by using:
 
 ```text
-usage: cargo run --all-features -- [Hex|B10] [legacy|kimchi] <OUTPUT_FILE>
+usage: cargo run -p export_test_vectors -- [Hex|B10] [legacy|kimchi] <OUTPUT_FILE>
```

Comment:
can you remove `usage:` here? I know you didn't add it, don't know why it was there in the first place :o